### PR TITLE
Check for errors from BrowserStack API on killCurrentSession

### PR DESCRIPTION
**Pull request by seriallos**
In my own usage of Chimp + Bamboo CI + BrowserStack, I've been seeing rare failures that I believe are related to this chunk of code. From time to time, the builds data that comes back is just an empty array which fails the `buildId !== ""` logic since `buildId` is actually `undefined`, not an empty string.

### CODE REVIEW

File: src/lib/browserstack-manager.js [hunk 1]
```diff
@@ -93,31 +93,39 @@ BrowserStackSessionManager.prototype._getSessions = function (buildId, callback)
...
-              callback();
-            } else {
-              log.error('[chimp][browserstack-session-manager]', 'received error', error);
-              callback(error);
-            }
-          });
-        }
-      }.bind(this));
+    if (err) {
+      log.error('[chimp][browserstack-session-manager]', 'received getBuilds error', err);
```
Comment (lucetius) [inline src/lib/browserstack-manager.js]: `err` is not needed here since you pass it to callback

File: src/lib/browserstack-manager.js [hunk 2]
```diff
@@ -93,31 +93,39 @@ BrowserStackSessionManager.prototype._getSessions = function (buildId, callback)
...
+      callback(err);
+    } else {
+      if (builds && builds.length) {
+        log.debug('[chimp][browserstack-session-manager]', builds, builds[0]);
+        var buildId = builds[0].automation_build.hashed_id;
+      }
+      if (buildId !== undefined && buildId !== '') {
```
Comment (lucetius) [inline src/lib/browserstack-manager.js]: I think `if (buildId)` should be enought

Comment (seriallos) [inline src/lib/browserstack-manager.js]: Fixed the comments. Thanks!

**Comment by seriallos** [comment]
Looks like this was addressed in other changes a couple months ago!

**Pull request closed without merging**